src/terminal.rs

```diff
@@ -149,6 +155,25 @@ impl TerminalCapabilities {
         }
     }
 
+    // Wez's Terminal Emulator
+    //
+    // WezTerm is a GPU-accelerated cross-platform
+    // terminal emulator and multiplexer written by @wez
+    // and implemented in Rust
+    //
+    // https://wezfurlong.org/wezterm/
+    //
+    /// Terminal capabilities of WezTerm
```

Comment:
Why is most of this not a doc comment?

Please do make a proper doc comment out of all this description.

Reply:
Is the current version okay?

Reply:
Thanks.